run_simfit_data_fullAngularMass_Swave.sh

```diff
@@ -14,6 +14,9 @@ bin=${2}
 
 yearConf=${3}
 
+XGBv=8
+fitopt=1
```

Comment:
I'd suggest to use a name for this variable that is a bit more self-explaining (both here and in the actual code). e.g. something like useBernsteinBkg or anything similar

Reply:
The original code has different fitoption depending on the bin choice: for bin4 0=exp; 1=Bernstein, for bin6 0=noZ4430, 1=yesZ4430, 2=yesZ4430+mean of the Z4430 contribution free. So, it is not related to Bernstein only

Reply:
ah ok. in the current one the expo to bernstein is the only modification I see